### PR TITLE
Campaign page v2

### PR DESCRIPTION
**Pull request by val-fcn**

### CODE REVIEW

File: frontend/src/components/CampaignCard.jsx [hunk 1]
```diff
@@ -2,51 +2,58 @@ import React, { useState, useEffect } from "react";
...
   }
 
+  useEffect(() => {
+    checkDate();
+  }, [timeDiff, dateDiff]);
+
   return (
-    <div className={`${cardStyle} related self`}>
-      <Link to={`/campaign/${campaignInfo.id}`}>
+    <div className={cardStyle}>
```
Comment (BastienSaulnier) [inline frontend/src/components/CampaignCard.jsx]: <div className={cardStyle}> need to be renamed to className={`${cardStyle} related self`}>

File: frontend/src/components/CampaignCard.jsx [hunk 2]
```diff
@@ -2,51 +2,58 @@ import React, { useState, useEffect } from "react";
...
+      <Link to={`/campaign/${id}`}>
```
Comment (BastienSaulnier) [inline frontend/src/components/CampaignCard.jsx]: <Link to={`/campaign/${id}`}> need to be set to <Link to={`/campaigns/${id}`}>

**Review by BastienSaulnier — DISMISSED** [review]
SnE

File: frontend/src/pages/CampaignPageById.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          className="profileImage"
+          src={campaignInfoById.img}
+          alt={campaignInfoById.name}
+        />
+        ​
+        <div className="infoContainer">
+          <div className="firstContainer">
+            <h2>{campaignInfoById.name}</h2>
+            <div className="profileTag">
+              <img src="https://via.placeholder.com/50x50" alt="Tag" />
```
Comment (LoicBrassart) [inline frontend/src/pages/CampaignPageById.jsx]: DAMNIT, NO !

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Just find your tag pictures and this PR shall pass

**Review by BastienSaulnier — APPROVED** [review]
SnE cause placeholder page is now suppressed

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**